Restructure the Guides section into journey-ordered story pages (#1134)
midnightntwrk/midnight-docs: docs/guides/development-environment.mdx

---
SPDX-License-Identifier: Apache-2.0
copyright: This file is part of midnight-docs. Copyright (C) Midnight Foundation. Licensed under the Apache License, Version 2.0 (the "License"); You may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
title: Set up your Midnight development environment

> description: Install Docker, the Compact compiler, the VS Code extension, and a local proof server on macOS, Linux, or Windows (WSL), and optionally set up Bun as your JavaScript runtime.

Author: Consider trimming the description under the 160 character guideline (currently 174).

```suggestion
description: Install Docker, the Compact compiler, the VS Code extension, and a local proof server on macOS, Linux, or Windows (WSL), with optional Bun setup.
```
